### DISCUSSION TITLE
How do I configure this extension?

### DISCUSSION THREAD

**Discussion by agronholm** [brucepro/Memoir#82] [Q&A] [answered]
The README lists a bunch of configuration options, but there doesn't seem to be any UI for these settings. How am I supposed to configure it?

**Comment by brucepro** [comment]
Should be at the bottom of the chat window once you enable it in session.

**Reply by agronholm** [reply]
Where?
![kuva](https://github.com/user-attachments/assets/5d1301bd-2a76-4e38-a254-37e8804301ae)
I do have it enabled:
![kuva](https://github.com/user-attachments/assets/1d8fdd0c-88ad-45e3-aa1a-ba5264380ac7)

**Reply by brucepro** [reply, marked as answer]
Should be below the chat box, what browser are you using?

**Reply by agronholm** [reply]
Firefox

**Reply by agronholm** [reply]
My bad, I didn't realize the window could be scrolled down. The design totally looked like it was meant to fill the entire screen.

**Reply by agronholm** [reply]
Thanks for the pointer though!

**Reply by brucepro** [reply]
Also what does the code line say at startup? Any errors? Maybe the extension failed to load.

**Reply by agronholm** [reply]
I think it's fine; I found this:
![kuva](https://github.com/user-attachments/assets/ee308740-6e8e-43e2-acf2-0c815d5a0503)

**Reply by agronholm** [reply]
I did however see these warnings at startup:
```
home/alex/ai/text-generation-webui/venv-textgen/lib/python3.12/site-packages/pydantic/_internal/_fields.py:132: UserWarning: Field "model_name" in ModelInfoResponse has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
  warnings.warn(
/home/alex/ai/text-generation-webui/venv-textgen/lib/python3.12/site-packages/pydantic/_internal/_fields.py:132: UserWarning: Field "model_names" in ModelListResponse has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
  warnings.warn(
/home/alex/ai/text-generation-webui/venv-textgen/lib/python3.12/site-packages/pydantic/_internal/_fields.py:132: UserWarning: Field "model_name" in LoadModelRequest has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
  warnings.warn(
```

**Reply by brucepro** [reply]
That is a normal warning from textgen when you have the interface open with no model loaded. Doesn't hurt anything.
